auth/src/main/java/com/firebase/ui/auth/ui/accountlink/WelcomeBackPasswordPrompt.java

```diff
@@ -122,29 +128,32 @@ private void next(final String email, final String password) {
                 .addOnSuccessListener(new OnSuccessListener<AuthResult>() {
                     @Override
                     public void onSuccess(AuthResult authResult) {
-                        // Get the social AuthCredential from the IDPResponse object, link
-                        // it to the email/password account.
                         AuthCredential authCredential =
                                 AuthCredentialHelper.getAuthCredential(mIdpResponse);
-                        authResult.getUser().linkWithCredential(authCredential);
-                        firebaseAuth.signOut();
```

Comment:
Humm, so should I revert and use a continuation or leave it at `linkWithCredential`?